app/notifications/new_aircraft_created_notification.rb

```diff
@@ -1,8 +1,5 @@
 class NewAircraftCreatedNotification < Noticed::Base
 
-  deliver_by :database, debug: true
```

Comment:
This was leftover from my testing on an earlier PR - may be why we saw that duplicate notification 'bug'. If so, I'll go close it.